exporter/containerimage/opts.go

```diff
@@ -49,7 +48,7 @@ func (c *ImageCommitOpts) Load(ctx context.Context, opt map[string]string) (map[
 		case exptypes.OptKeyName:
 			c.ImageName = v
 		case exptypes.OptKeyOCITypes:
-			err = parseBoolWithDefault(&c.OCITypes, k, v, true)
```

Comment:
Not handling the empty value in here seems like possibly breaking change. Don't remember the history behind it but maybe don't break it as it seems harmeless.

Reply:
That change was made for the other types that originally used this parsing function, so based on that I'm relatively confident in/in favor of this change.